src/validator.js

```diff
@@ -231,6 +235,18 @@ export default class Validator {
         })
         .then(() => {
           this.xpi = new _Xpi(this.packagePath);
+          return this.xpi.getMetaData();
+        })
+        .then((metadata) => {
+          var chromeManifest = 'chrome.manifest';
+          if (Object.keys(metadata).indexOf(chromeManifest) > -1) {
+            return this.scanFile(chromeManifest, 'stream');
+          } else {
+            // TODO: Log this.
```

Comment:
I'm not yet sure about that, it doesn't look like it looking at some of the code in the original [1]

https://github.com/mozilla/amo-validator/blob/master/tests/test_chromemanifest_testcases.py#L19